### DISCUSSION TITLE
Custom Tic-80 folder

### DISCUSSION THREAD

**Discussion by bettywolfy** [nesbox/TIC-80#2378] [Ideas]
Is it possible to create a system for selecting the tic-80 folder on devices like Android and iOS?
![Screenshot_20231126_113520_Files](https://github.com/nesbox/TIC-80/assets/92587199/dfbab4a7-fd1d-418b-8336-b2989f1f1360)

**Comment by JustaLiriK** [comment]
I've got the same issue, android seems to refuse access to the android/data/ folder
And tic-80 terminal don't allow to navigate outside tic-80's installation folder.
Did you succeed?
